gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/rebind/serialization/JacksonMapperProviderGenerator.java

```diff
@@ -110,12 +92,16 @@ protected String getTemplate() {
 
     @Override
     protected String getPackageName() {
-        return JacksonMapperProvider.class.getPackage().getName();
+        return PACKAGE;
     }
 
     @Override
     protected String getImplName() {
-        return JacksonMapperProvider.class.getSimpleName() + IMPL;
+        return CLASS_NAME;
+    }
+
+    void addDefinition(SerializationDefinition definition) {
```

Comment:
No. Being used by JacksonMapperGenerator (same package). I'll pull up to an interface.